org.eclipse.xtext/src/org/eclipse/xtext/service/OperationCanceledError.java

```diff
@@ -10,14 +10,13 @@
 /**
  * @since 2.8
  */
-@SuppressWarnings("all")
+@SuppressWarnings("serial")
```

Comment:
Is this about the missing serialization id? Might be better to generate one then

Reply:
we don't serialize anything, but why not. I've added it.